apiserver/facades/client/imagemetadatamanager/metadata.go

```diff
@@ -0,0 +1,177 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package imagemetadatamanager
+
+import (
+	"sort"
+
+	"github.com/juju/errors"
+	"github.com/juju/loggo"
+
+	"github.com/juju/juju/apiserver/common"
+	"github.com/juju/juju/apiserver/common/imagecommon"
+	"github.com/juju/juju/apiserver/facade"
+	"github.com/juju/juju/apiserver/params"
+	"github.com/juju/juju/environs"
+	"github.com/juju/juju/permission"
+	"github.com/juju/juju/state"
+	"github.com/juju/juju/state/cloudimagemetadata"
+	"github.com/juju/juju/state/stateenvirons"
+)
+
+var logger = loggo.GetLogger("juju.apiserver.imagemetadatamanager")
+
+// API is the concrete implementation of the api end point
+// for loud image metadata manipulations.
+type API struct {
+	metadata   metadataAccess
+	newEnviron func() (environs.Environ, error)
+	authorizer facade.Authorizer
```

Comment:
Yes, true now that they are split there is no need. It also means that we no longer need to have ControllerTag call - a lot cleaner code. Done.